conn.go

```diff
@@ -87,7 +87,7 @@ const (
 type Conn struct {
 	// requestTimeout is loaded atomically
 	// so we need to ensure 64-bit alignment on 32-bit platforms.
-	requestTimeout      int64
+	requestTimeout      time.Duration
```

Comment:
If we're no-longer loading requestTimeout atomically, we can probably remove the comment.

Has it been established that we don't need to read/write this field atomically? (I'm curious why it was set up that way before)

Reply:
I overlooked the comment, thanks for pointing out! Looking at the commit history, this was added in 412981c9 by Psychotropos. After reading through the comments of PR #199  and the linked issues this still seems to be a problem for the supported Go versions. I'll revert this part and stick with int64 and atomic loading to ensure 64-bit alignment. 

I thought I can get rid of the "unnecessary" type conversion 😅